### PR TITLE
silence systemctl (daemon-)reload

### LINKED ISSUE #1002: systemd errors in CI
See https://travis-ci.com/github/openhab/openhabian/jobs/353508710#L1186

### PR DESCRIPTION
**Pull request by mstormi**
also dont add openhab user to gpio group twice

Fixes #1002

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

**Comment by mstormi** [comment]
So Ethan bad luck for you: I was able to select you as a reviewer now

**Comment by ecdye** [comment]
Glad that it works. They are silenced with `cond_redirect` and it is intentional that the output is displayed in this case. I believe there is a quiet option for `update-alternatives` but I don't know if it can be set globally.

File: functions/openhab.bash [hunk 1]
```diff
@@ -68,8 +68,7 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   openhabVersion="$(apt-cache madison openhab2 | head -n 1 | cut -d'|' -f2 | xargs)"
 
   if ! cond_redirect apt-get -y --allow-downgrades install "openhab2=${openhabVersion}"; then echo "FAILED (apt)"; return 1; fi
-  cond_redirect adduser openhab gpio
```
Comment (ecdye) [inline functions/openhab.bash:71]: Why remove?

Comment (mstormi) [reply functions/openhab.bash:71]: it fails because it's done in some other place before

**Pull request merged**